src/ctapipe/tools/calculate_pixel_stats.py

```diff
@@ -40,16 +38,6 @@ class PixelStatisticsCalculatorTool(Tool):
 
     """
 
-    allowed_tels = Set(
```

Comment:
Why remove the `allowed_tels` option here?

Reply:
I will add this option back based on internal discussion